### PR TITLE
Adds filled-in variants for thumbsup and thumbsdown icons.

### LINKED ISSUE #161: Filled in variants of thumbsup/thumbsdown
**Is your icon request related to a problem? Please describe.**
Use case is that I'd like to use thumbsup/thumbsdown for voting buttons, and I'd like to use filled variants to show that your vote has been registered.

**Describe the solution you'd like**
Same as the existing thumbsup/thumbsdown, just filled in.

**Describe alternatives you've considered**
Forking the lib to add the new options.

### PR DESCRIPTION
**Pull request by drob**
These are handy if you want to indicate to a user that a thumbs up or thumbs down action has been registered.

Fixes #161 and #163

Screenshot 2023-02-02 at 6 59 23 PM

### CODE REVIEW

**Comment by drob** [comment]
@microsoft-github-policy-service agree company="qqbot"

**Comment by drob** [comment]
@daviddossett bumped!

File: src/template/mapping.json [hunk 1]
```diff
@@ -348,7 +348,9 @@
   "text-size": 60265,
   "three-bars": 60266,
   "thumbsdown": 60267,
+  "thumbsdown-filled": 60267,
```
Comment (daviddossett) [inline src/template/mapping.json:351]: These should go at the very bottom of the list and use unique values. I just add 1 to whatever the last icon was. So in this case:

```
"mic": 60434,
"thumbsdown-filled": 60435,
"thumbsup-filled": 60436
```

Then rerun the build task and you should be good to go

**Review by daviddossett — CHANGES_REQUESTED** [review]

File: src/template/mapping.json (hunk 1 above)
Comment (drob) [reply src/template/mapping.json]: Fixed!

**Review by daviddossett — APPROVED** [review]

**Review by rzhao271 — APPROVED** [review]

**Pull request merged**